nautobot_device_lifecycle_mgmt/models.py

```diff
@@ -542,6 +739,7 @@ class VulnerabilityLCM(PrimaryModel):
 
     cve = models.ForeignKey(CVELCM, on_delete=models.CASCADE, blank=True, null=True)
     software = models.ForeignKey(to="dcim.SoftwareVersion", on_delete=models.CASCADE, blank=True, null=True)
+    old_software = models.ForeignKey(SoftwareLCM, on_delete=models.CASCADE, blank=True, null=True)
```

Comment:
Same as above:

```suggestion
    old_software = models.ForeignKey(SoftwareLCM, on_delete=models.SET_NULL, blank=True, null=True)
```

Reply:
Agreed. Let's set it to `models.SET_NULL`